### PR TITLE
Add Alert Overrides Blog Post

### PR DESCRIPTION
**Pull request by kingthorin**
I finally got around to doing this. zapbot.png is a duplicate of the one from the Jan 2016 newsletter, I figured duplicating was the best option maintenance wise. I'm open to other suggestions for the image or duplication.

### CODE REVIEW

**Comment by kingthorin** [comment]
Here's a Firefox fullpage screenshot:
image

File: site/content/blog/2020-05-10-alert-overrides-youdontknowzap/index.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+title: "Customize Alert Details - #YouDontKnowZAP"
+description: "ZAP functionality that let's you customize Alert details"
+images:
+- https://www.zaproxy.org/blog/2020-05-10-alert-overrides-youdontknowzap/images/zapbotpng
```
Comment (thc202) [inline site/content/blog/2020-05-10-alert-overrides-youdontknowzap/index.md:5]: Missing dot in the file name.

File: site/content/blog/2020-05-10-alert-overrides-youdontknowzap/index.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+- youdontknowzap
+date: "2020-05-10"
+addSocialPreview: true
+authors:
+    - thorin
+---
+
+Did you know that you or your company/organization could customize the generic details of the alerts that ZAP raises?
+
+Alerts raised by ZAP contain a variety of information, some generic, some specific to the issue at hand. Specific details may includes things such as
```
Comment (thc202) [inline site/content/blog/2020-05-10-alert-overrides-youdontknowzap/index.md:18]: `include`? (Same for following sentence.)

File: site/content/blog/2020-05-10-alert-overrides-youdontknowzap/index.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+For example:
+
+```text
+# 40012 = Reflected XSS
+40012.solution = Follow our company specific guidelines at http://internet.example.com/xss.html
+```
+
+## Example Usage
+
+Let's say you work for Juice Shop coporation, and the organizaiton is particular sensitive to disclosure of internal IP addresses. After an initial scan with ZAP
```
Comment (thc202) [inline site/content/blog/2020-05-10-alert-overrides-youdontknowzap/index.md:50]: Typo in `corporation` and `organization`.

File: site/content/blog/2020-05-10-alert-overrides-youdontknowzap/index.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+2.reference = +\nhttp://intranet.owasp-juice.shop/devdocs/internal-ip/
+# 10096 - Timestamp Disclosure
+10096.solution = If this timestamp evaluates to a date before 2014-10-04 (First release) then the finding is almost definitely a false positive.
+```
+
+This file can be leveraged by:
+
+- Specifying its use (location) in ZAP's Options dialog.
+- Via the [API](https://www.zaproxy.org/docs/desktop/start/features/api/)
+- From the commandline: `-config alert.overridesFilename=/path/to/filename`
```
Comment (thc202) [inline site/content/blog/2020-05-10-alert-overrides-youdontknowzap/index.md:68]: `command line`?

File: site/content/blog/2020-05-10-alert-overrides-youdontknowzap/index.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+
+Setting the file via the Options dialog:
+![Options Dialog](images/options-dialog.png)
+
+Overridden details:
+![IOverridden Results](images/overridden-results.png)
```
Comment (thc202) [inline site/content/blog/2020-05-10-alert-overrides-youdontknowzap/index.md:74]: Remove `I` in `Overridden`?

File: site/content/blog/2020-05-10-alert-overrides-youdontknowzap/index.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+# 2 - Private IP Disclosure
+2.reference = +\nhttp://intranet.owasp-juice.shop/devdocs/internal-ip/
+# 10096 - Timestamp Disclosure
+10096.solution = If this timestamp evaluates to a date before 2014-10-04 (First release) then the finding is almost definitely a false positive.
+```
+
+This file can be leveraged by:
+
+- Specifying its use (location) in ZAP's Options dialog.
+- Via the [API](https://www.zaproxy.org/docs/desktop/start/features/api/)
```
Comment (thc202) [inline site/content/blog/2020-05-10-alert-overrides-youdontknowzap/index.md:67]: Could use relative path.

File: site/content/blog/2020-05-10-alert-overrides-youdontknowzap/index.md [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+
+Setting the file via the Options dialog:
+![Options Dialog](images/options-dialog.png)
+
+Overridden details:
+![IOverridden Results](images/overridden-results.png)
+<br>
+![After Override](images/after-override.png)
+
+For further details see this [help entry](https://www.zaproxy.org/docs/desktop/start/features/alerts/#alert-overrides).
```
Comment (thc202) [inline site/content/blog/2020-05-10-alert-overrides-youdontknowzap/index.md:78]: Could use relative path.

**Comment by kingthorin** [comment]
Thanks, will tackle all that shortly.

**Comment by kingthorin** [comment]
Fixed up.

File: site/content/blog/2020-05-10-alert-overrides-youdontknowzap/index.md [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+- Specifying its use (location) in ZAP's Options dialog.
+- Via the [API](/docs/desktop/start/features/api/)
+- From the command line: `-config alert.overridesFilename=/path/to/filename`
+
+Setting the file via the Options dialog:
+![Options Dialog](images/options-dialog.png)
+
+Overridden details:
+![Overridden Results](images/overridden-results.png)
+<br>
```
Comment (thc202) [inline site/content/blog/2020-05-10-alert-overrides-youdontknowzap/index.md:75]: This needs to be tweaked, HTML tags are removed (actually replaced with `<!-- raw HTML omitted -->`).

Comment (kingthorin) [reply site/content/blog/2020-05-10-alert-overrides-youdontknowzap/index.md:75]: Okay

**Comment by kingthorin** [comment]
Fixed.

**Comment by thc202** [comment]
Thanks!

**Pull request merged**